content/en/docs/concepts/storage/volumes.md

```diff
@@ -1155,6 +1155,21 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+The CSI Migration feature for vsphereVolume, when enabled, shims all plugin operations
+from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}} Driver. In order to use this feature, the [vSphere CSI
+Driver](https://github.com/kubernetes-sigs/vsphere-csi-driver)
+must be installed on the cluster and the `CSIMigration` and `CSIMigrationvSphere`
+features must be enabled.
+
+#### CSI Migration Complete
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+To turn off the vsphereVolume plugin from being loaded by controller manager and kubelet, you need to set this feature flag to true. This requires `csi.vsphere.vmware.com` {{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}} driver being installed on all worker nodes.
```

Comment:
I'm not sure what this section is explaining.

Reply:
This actually follows the similar doc section from the Azure driver.
It means that when the "CSI Migration Complete" is set true, CSI migration will stop working.  This requires CSI driver to be installed on all nodes.